docs/reconciliation.md

```diff
@@ -5,22 +5,13 @@ Once a composition has been synthesized, the resulting resources are reconciled
 
 ## Merge Semantics
 
-Eno uses the [structured-merge-diff](https://pkg.go.dev/sigs.k8s.io/structured-merge-diff) package to merge the expected and actual states of each resource using rules defined by its schema.
+By default, Eno uses [server-side apply](https://kubernetes.io/docs/reference/using-api/server-side-apply/) to create and update resources it manages.
 
 Any property set by the synthesizer will be applied during reconciliation, even if it means overwriting changes made by another client.
 Other clients can safely populate fields _not_ managed by Eno - unmanaged fields are not modified.
+This is the standard behavior of server-side apply with `--force-conflicts=true`.
```

Comment:
How does this impact future work on "partially managed" fileds?

Reply:
Not too much I think. The reconciler still has all of the state it needs to compute the diffs - we will just need to implement that patch logic ourselves or using jsonpatch rather than tieing into the fieldpath bits of structured-merge-diff.